airbyte-ci/connectors/metadata_service/lib/README.md

```diff
@@ -34,3 +34,17 @@ poetry run pytest
 ```bash
 poetry run metadata_service validate tests/fixtures/valid/metadata_registry_override.yaml
 ```
+
+## Useful Commands
+
+### Replicate Production Data in your Development Bucket
+This will replicate all the production data to your development bucket. This is useful for testing the metadata service with real up to date data.
```

Comment:
What happens if there is already data in my dev instance? Will it overwrite but not remove? Full refresh? (omg etl returns)

Reply:
It will fully overwrite and remove any files in your destination that are not at your source.

Full replication

Reply:
Sounds expected to me - thanks for adding a note to the docs!